respecConfig.json

```diff
@@ -1,9 +1,9 @@
 {
-  "specStatus": "unofficial",
+  "specStatus": "base",
```

Comment:
Is there a plan to publish docs like this?
https://respec.org/docs/#specStatus

Reply:
I see the issuer repo is using `unofficial`.  This should be the same one way or another.
https://github.com/w3c-ccg/vc-api-issuer-test-suite/pull/57

Reply:
That should not have changed...actually. This is still a CCG test suite, so should stay "unofficial".